From 1f02da5c8cc9dc711ded015059ae85a5126dd269 Mon Sep 17 00:00:00 2001
From: Andrew Pogrebennyk <apogrebennyk@sipwise.com>
Date: Thu, 29 Dec 2022 10:14:12 +0100
Subject: MT#55831 increase the config line length to 4096 because of header_list

increase the config line length to 4096 because of header_list
(real ticket number: MT#1640)

--- a/core/AmConfigReader.h
+++ b/core/AmConfigReader.h
@@ -33,7 +33,7 @@
 using std::string;
 
 
-#define MAX_CONFIG_LINE 2048
+#define MAX_CONFIG_LINE 4096
 #define CONFIG_FILE_SUFFIX ".conf"
 
 /**
